op-deployer/pkg/deployer/manage/add_game_type_test.go

```diff
@@ -25,7 +25,9 @@ import (
 
 func TestAddGameType(t *testing.T) {
 	rpcURL := os.Getenv("SEPOLIA_RPC_URL")
-	require.NotEmpty(t, rpcURL, "must specify RPC url via SEPOLIA_RPC_URL env var")
+	if rpcURL == "" {
+		t.Skip("Skipping test, SEPOLIA_RPC_URL not set")
```

Comment:
Similar to the comment in `implementations_test.go`, I think it's better to speficy that the RPC URL is missing.